Makefile

```diff
@@ -32,18 +32,18 @@ build-latest:
 
 build-testing:
 	$(MAKE) build TAG=testing \
-	    PROTOCOL_VERSION_DEFAULT=21 \
-		XDR_REF=v21.1.0 \
-		CORE_REF=v21.3.1 \
+	    PROTOCOL_VERSION_DEFAULT=22 \
```

Comment:
Should protocol version on testing target be set to 21 until upgrade vote of testnet? Wondering if anyone might use this quickstart image to connect to Testnet.

Reply:
+1 it should stay 21 until we have a horizon and rpc.

Reply:
I believe this parameter only affects when you're running quickstart as a local network.

Reply:
Ah good point about horizon and rpc. I'll revert.